test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -9643,6 +9643,100 @@ public async Task RestoreNetCore_CPVMProject_AddRemoveNotProjectRelatedEntriesTo
             }
         }
 
+        /// <summary>
+        /// Project A -> PackageX 100
+        ///           -> PackageY 200 -> PackageX 200
+        ///            -> ProjectB -> ProjectC -> PackageX 100
+        ///  All projects CPVM enabled; PackageX 100 and PackageY 200 in cpvm file         
+        ///  Expected NU1605                     
+        /// </summary>
+        [Fact]
+        public async Task RestoreCommand_DowngradeIsNotErrorWhen_DowngradedByCentralDirectDependency_P2P()
```

Comment:
Similar as the other method. 

Try: 
`MethodName_StateUnderTest_ExpectedBehavior`

We need some guides around this.
Bumped the priority of https://github.com/NuGet/Client.Engineering/issues/132. It might be one of the tasks I pick up soon.

Reply:
Changed the name